app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -20,25 +20,30 @@
 
 <%= render 'nav', step_by_step_page: @step_by_step_page, active: "internal-change-notes" %>
 
+<%= render "govuk_publishing_components/components/contextual_guidance", {
+  html_for: "internal-change-note-description",
+  title: t("step_by_step_pages.internal_change_notes.label"),
+  content: t("step_by_step_pages.internal_change_notes.guidance")
+} do %>
```

Comment:
Would this normally be inside the form and wrap just the textarea?

Reply:
Yes, it would normally, but since we only have one input I thought I can leave it close to the textarea and submit to make it clear there are not other inputs in the page. Let me know if it's a blocker though, I can go through them and update the nesting order.

Reply:
Hmmm I'm with Kevin on this one I think. If someone wants to add another input to this page in the future, they'd have to re-nest everything appropriately so that this contextual guidance doesn't apply to their input. I don't think there's much value in making it clear there's currently only one input on the page.